google/cloud/firestore_admin_v1/services/firestore_admin/async_client.py

```diff
@@ -249,7 +249,7 @@ def transport(self) -> FirestoreAdminTransport:
         return self._client.transport
 
     @property
-    def api_endpoint(self):
+    def api_endpoint(self) -> str:
```

Comment:
The files under `google/cloud/firestore_admin_v1/services` is an autogenerated file, so this won't persist. (I opened a request to fix it in the auto-generator though)

Please revert these for now

Reply:
Ah got it. Reverted.